src/command/info/info.c

```diff
@@ -140,10 +287,11 @@ archiveDbList(const String *stanza, const InfoPgData *pgData, VariantList *archi
     String *archiveStart = NULL;
     String *archiveStop = NULL;
     Variant *archiveInfo = varNewKv(kvNew());
+    const Storage *storageRepo = storageRepoIdx(repoIdx);
```

Comment:
Not sure we need to create these storageRepo variables. I think using storageRepoIdx(repoIdx) in place is clearer and there aren't so many storage calls that it will make any difference performance-wise.